src/sagas.js

```diff
@@ -397,7 +445,15 @@ function* fetchConsoleMetadatas ({ vms }) {
   })
 }
 
-function* fetchVmDisks ({ vmId }) {
+export function* fetchDisks ({ vms }) {
+  yield * foreach(vms, function* (vm) {
+    const vmId = vm.id
+    const disks = yield fetchVmDisks({ vmId })
+    yield put(setVmDisks({ vmId, disks }))
+  })
+}
+
+function* fetchVmDisks ({ vmId }) { // Would __additional__ (API param __follow_) work here as well?
```

Comment:
AFAIK `follow` parameter works for any subresource starting engine 4.2, http://ovirt.github.io/ovirt-engine-api-model/master/#documents/003_common_concepts/follow

Reply:
Agreed.  I'll adjust the comment to list is as a TODO.  It can be addressed in the future since it works now but could work much better.